csaf/schema/ROLIE_feed_json_schema.json

```diff
@@ -36,7 +36,7 @@
           }
         }
       ],
-      "minItems": 1,
+      "minItems": 0,
```

Comment:
Please use the latest version of the schema from https://github.com/oasis-tcs/csaf/pull/481/files

Reply:
Adressed by  https://github.com/csaf-poc/csaf_distribution/pull/357/commits/71de55f7c1ed5f3a579a406d075693023f08bd7c